docs/postpaid/response-code.md

```diff
@@ -2,52 +2,52 @@
+01 | INVOICE HAS BEEN PAID | Failed | The invoice with your inputted data has already been paid. You don’t need to pay it again, or you can check for your inputted customer_id.
+02 | BILL UNPAID | Failed | Your bill is unpaid, only reaching inquiry status. Please finish your payment first.
+03 | INVALID REF ID | Failed | Your inputted ref_id isn’t valid. Please follow the correct format for ref_id (alpha_num only without space). Try again with a valid ref_id.
+04 | BILLING ID EXPIRED | Failed | Your reference ID (ref_id) is expired. Please retry your inquiry request with a different reference ID (ref_id).
+05 | UNDEFINED ERROR | Pending | Your transaction failed because of an undefined error. Please try again later.
+06 | INQUIRY ID NOT FOUND | Failed | The inquiry ID (tr_id) that you’ve inputted is not found, there is no inquiry with that ID. You can check the inquiry ID (tr_id) field for any typos, or try using another inquiry ID.
+07 | TRANSACTION FAILED | Failed | Your transaction has failed. Please try again.
+08 | BILLING ID BLOCKED | Failed | The customer ID for your inputted product code is blocked by IAK. Please contact our Customer Service.
+09 | INQUIRY FAILED | Failed | Your inquiry process failed. Please try to do the inquiry again.
+10 | BILL IS NOT AVAILABLE | Failed | The bill isn’t available yet. Please try again when the bill is already added/available.
+11 | DUPLICATE REF ID | Failed | The ref_id that you’ve inputted is already been inputted, try again with another ref_id.
+13 | CUSTOMER NUMBER BLOCKED | Failed | Your custom number (customer_id) has been blocked by Indobest Artha Kreasi. You can change your customer number (customer_id) or ask IAK’s Customer Service.
+14 | INCORRECT DESTINATION NUMBER | Failed | The customer/destination number that you’ve inputted is incorrect. Try checking your customer/destination number for typos or try with another customer/destination number.
+15 | NUMBER THAT YOU ENTERED IS NOT SUPPORTED | Failed | Customer number that you’ve inputted isn’t supported by your inputted product_code. Try again with another customer number or product_code.
+16 | NUMBER DOESN'T MATCH THE OPERATOR | Failed | Your customer number (customer_id) that you’ve inputted doesn’t match with your desired operator (product_code). Please check again your customer number or change your operator.
+17 | BALANCE NOT ENOUGH | Failed | Your current balance is lower than the product price you want to buy. You can add more money into your deposit by doing top up on iak.id deposit menu, or if you are in development mode, you can add your development deposit by clicking the + (plus) sign on development deposit menu (https://developer.iak.id/sandbox-report).
+20 | PRODUCT UNREGISTERED | Failed | Your inputted product_code isn’t in the database. Check again your product_code, you can check product_code list by using pricelist API (https://api.iak.id/docs/reference/ad7437ff00ecc-price-list).
```

Comment:
Your inputted code isn’t in the database. Check again your code, you can check code list by using pricelist API (https://api.iak.id/docs/reference/ad7437ff00ecc-price-list).